packages/kmod-6.1-neuron/kmod-6.1-neuron.spec

```
@@ -51,9 +58,19 @@ install -p -m 0644 %{S:1} %{buildroot}%{_cross_libdir}/modules-load.d/neuron.con
 install -d %{buildroot}%{_cross_unitdir}/systemd-modules-load.service.d
 install -p -m 0644 %{S:2} %{buildroot}%{_cross_unitdir}/systemd-modules-load.service.d/neuron.conf
 
+# Install the shared header file
+install -d %{buildroot}%{_cross_includedir}/kmod-6.1-neuron/share
+install -p -m 0644 %{_builddir}/%{neuron_sources}/share/neuron_driver_shared.h %{buildroot}/%{_cross_includedir}/kmod-6.1-neuron/share/neuron_driver_shared.h
+install -p -m 0644 %{_builddir}/%{neuron_sources}/neuron_ioctl.h %{buildroot}/%{_cross_includedir}/kmod-6.1-neuron/neuron_ioctl.h
+
 %files
 %license %{neuron_sources}/LICENSE
 %{_cross_attribution_file}
 %{_cross_libdir}/modules/*/extra/neuron.ko.gz
 %{_cross_libdir}/modules-load.d/neuron.conf
 %{_cross_unitdir}/systemd-modules-load.service.d/neuron.conf
+
+%files devel
+%dir %{_cross_includedir}/kmod-6.1-neuron
+%{_cross_includedir}/kmod-6.1-neuron/share/neuron_driver_shared.h
+%{_cross_includedir}/kmod-6.1-neuron/neuron_ioctl.h
```

Is this where downstream builds expect the headers?

You could put them in the normal include dirs:
```suggestion
%{_cross_includedir}/share/neuron_driver_shared.h
%{_cross_includedir}/neuron_ioctl.h
```

---

I'll move them, I didn't know if BR wants to separate headers by package or not.